main/src/cgeo/geocaching/models/Waypoint.java

```diff
@@ -142,11 +142,12 @@ private int order() {
         return cachedOrder;
     }
 
+    @NonNull
     public String getPrefix() {
         return prefix;
     }
 
-    public void setPrefix(final String prefix) {
+    public void setPrefix(final @NonNull String prefix) {
```

Comment:
I pushed a new commit fixing this. Did github send a notifiaction about adding a commit changing a existing merge request?